src/main/java/seedu/address/logic/parser/FilterCommandParser.java

```diff
@@ -37,37 +44,182 @@ public FilterCommand parse(String args) throws ParseException {
                 ArgumentTokenizer.tokenize(args, PREFIX_TIME, PREFIX_INGREDIENT_GRAIN, PREFIX_INGREDIENT_VEGE,
                         PREFIX_INGREDIENT_PROTEIN, PREFIX_INGREDIENT_OTHER, PREFIX_GOAL);
 
-        List<Time> filterByTime;
+        boolean filterByFavourites = filterByFavourites(argMultimap);
```

Comment:
It might be better to name it something along the lines of 'isFilteredByFavourites'!

Reply:
Thanks for the feedback. I agree with you and will make the change :)